Formula/p/pikchr.rb

```diff
@@ -0,0 +1,22 @@
+class Pikchr < Formula
+  desc "PIC-like markup language for diagrams in technical documentation"
+  homepage "https://pikchr.org/"
+  url "https://pikchr.org/home/zip/ae3317b0ec/pikchr-ae3317b0ec.zip"
```

Comment:
how does this url get derived?

Reply:
The upstream website has their own UI on top of Github.  They don't publish version numbers, so I fix the version with the commit id, and use the commit date in format YYYY-MM-DD as a version

Reply:
> They don't publish version numbers

that seems a flag to me, any reason why (or can you ask about it)?

Reply:
I left a question about this in their forum

Reply:
Hi Chen,

Thank you for the suggestion to ask upstream for a proper version number.  They agreed to tag the github repo with 1.0.0, so we have a good url and a good version now